src/data/ability.ts

```diff
@@ -5235,7 +5243,7 @@ export function initAbilities() {
     new Ability(Abilities.SHEER_FORCE, 5)
       .attr(MovePowerBoostAbAttr, (user, target, move) => move.chance >= 1, 5461 / 4096)
       .attr(MoveEffectChanceMultiplierAbAttr, 0)
-      .partial(),
+      .edgeCase(), // Does not work with Meloetta relic song, life orb, eject button, red card, shell bell, kee/maranga berry, color change, pickpocket, wimp out, emergency exit, berserk, anger shell
```

Comment:
```suggestion
      .edgeCase() // Does not disable shell bell or Meloetta's relic song transformation
      .edgeCase(), // Does not disable life orb, eject button, red card, kee/maranga berry (unimplemented items)
```
Sheer Force does correctly disable abilities like Pickpocket (e.g., see #4631)